src/screens/shared/farmProfile/FarmContactCard.js

```diff
@@ -11,15 +12,18 @@ class FarmContactCard extends React.PureComponent {
   render() {
     const { farm } = this.props;
     const {
-      farmAddress,
+      address: physicalAddress,
+      mailingAddress,
       contactFirstName,
       contactLastName,
       farmEmail: email,
-      GroupGapContactIds: inspector,
-      phone
+      phone,
+      foodHubAffiliation: foodHub
```

Comment:
handle case where foodHub is no, meaning there is no food hub affiliation yet 

phoebe is getting the details from spring about how to display this information